src/config/logger.ts

```diff
@@ -45,6 +46,13 @@ const errorSerializer = (err) => (!err || !err.stack) ? err : {
 	stack: getFullErrorStack(err)
 };
 
+const hashSerializer = (data) => {
+	if (!data) {
+		return data;
+	}
+	return createHashWithSharedSecret(data);
```

Comment:
Just for my understanding, we're encrypting all the values, not removing them, and we can still decrypt it if needed right?

We have a way of doing this easily?  Is this using cryptor?